src/constants.ts

```diff
@@ -17,6 +17,8 @@ export const POSTGRES_ADMIN_PASSWORD = process.env.POSTGRES_ADMIN_PASSWORD
 
 export const INVITATION_URL = process.env.INVITATION_URL
 
+export const USE_SOCKETDOCK = process.env.USE_SOCKETDOCK || 'false'
```

Comment:
This way we can use it as boolean

```suggestion
export const USE_SOCKETDOCK = process.env.USE_SOCKETDOCK === 'true'
```

Reply:
Thank you for the suggestion! I've updated the code .